vcr/issuer/openid4vci/issuer.go

```diff
@@ -376,3 +436,14 @@ func generateCode() string {
 	}
 	return base64.URLEncoding.EncodeToString(buf)
 }
+
+func deepcopy(src []map[string]interface{}) []map[string]interface{} {
```

Comment:
Not really a deep copy (just of the keys), could use `mohae/deepcopy.Copy()` which is already a (transitive?) dependency

Reply:
and the slice.
I didn't want to use that dep, not really needed here.